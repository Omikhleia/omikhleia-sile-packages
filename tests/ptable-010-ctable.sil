\begin[papersize=a6]{document}
\nofolios
\script[src=packages/ptable]
\script[src=packages/raiselower]
% Table with cell border configuration at cell level:
% This table example is in the LaTeX ctable package, let us try to reproduce it.

\define[command=mysuper]{\raise[height=0.77ex]{\font[size=1.414ex]{\process}}}

\center{\parbox[width=80%fw, strut=character]{%
\begin[cols=20%fw 40%fw 40%fw, cellborder=0]{ptable}
  \begin{row}
    \cell[border=0.8pt 0.4pt 0 0]{}
    \cell[border=0.8pt 0.4pt 0 0, style=center]{H(Mu) + F\mysuper{2}}
    \cell[border=0.8pt 0.4pt 0 0, style=center]{H(Mu) + Cl\mysuper{2}}
  \end{row}
  \begin{row}
    \cell{\em{β}\kern[width=0.1em](H)}
    \cell[style=center]{80.9°}
    \cell[style=center]{83.2°}
  \end{row}
  \begin{row}
    \cell[border=0 0.8pt 0 0]{\em{β}\kern[width=0.1em](Mu)}
    \cell[border=0 0.8pt 0 0, style=center]{86.7°}
    \cell[border=0 0.8pt 0 0, style=center]{87.7°}
  \end{row}
\end{ptable}
}}

% Again done differently.

\center{%
\begin[cols=16%fw 32%fw 32%fw, cellborder=0]{ptable}
  \begin{row}
    \cell[border=0.8pt 0.4pt 0 0]{}
    \cell[border=0.8pt 0.4pt 0 0, style=center]{H(Mu) + F\mysuper{2}}
    \cell[border=0.8pt 0.4pt 0 0, style=center]{H(Mu) + Cl\mysuper{2}}
  \end{row}
  \begin{row}
    \cell{\em{β}\kern[width=0.1em](H)}
    \cell[style=center]{80.9°}
    \cell[style=center]{83.2°}
  \end{row}
  \begin{row}
    \cell[border=0 0.8pt 0 0]{\em{β}\kern[width=0.1em](Mu)}
    \cell[border=0 0.8pt 0 0, style=center]{86.7°}
    \cell[border=0 0.8pt 0 0, style=center]{87.7°}
  \end{row}
\end{ptable}
}

\end{document}