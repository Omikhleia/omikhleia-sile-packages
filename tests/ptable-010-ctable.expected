Set paper size 	595.275597	841.8897729
Begin page
Mx 	190.4882
My 	60.2488
Draw line	190.4882	48.0945	42.8598	0.8000
Draw line	190.4882	66.6902	42.8598	0.4000
Draw line	233.3480	48.0945	85.7197	0.8000
Draw line	233.3480	66.6902	85.7197	0.4000
Mx 	257.3480
Set font 	Gentium Plus;10;400;;normal;;LTR
T	43 w=6.5381	(H)
Mx 	263.8861
T	11 w=3.1689	(()
Mx 	267.0551
T	48 88 w=13.3594	(Mu)
Mx 	280.4144
T	12 w=3.1689	())
Mx 	286.2275
T	14 w=4.1016	(+)
Mx 	292.9733
T	41 w=4.7803	(F)
Mx 	297.7535
My 	56.7522
Set font 	Gentium Plus;6.421;400;;normal;;LTR
T	21 w=3.0130	(2)
Draw line	319.0677	48.0945	85.7197	0.8000
Draw line	319.0677	66.6902	85.7197	0.4000
Mx 	343.0677
My 	60.2488
Set font 	Gentium Plus;10;400;;normal;;LTR
T	43 w=6.5381	(H)
Mx 	349.6058
T	11 w=3.1689	(()
Mx 	352.7748
T	48 88 w=13.3594	(Mu)
Mx 	366.1341
T	12 w=3.1689	())
Mx 	371.9469
T	14 w=4.1016	(+)
Mx 	378.6922
T	38 79 w=8.0713	(Cl)
Mx 	386.7635
My 	56.7522
Set font 	Gentium Plus;6.421;400;;normal;;LTR
T	21 w=3.0130	(2)
Mx 	190.4882
My 	78.8445
Mx 	214.4882
Set font 	Gentium Plus;10;400;Italic;normal;;LTR
T	478 w=4.6289	(β)
Mx 	220.1171
Set font 	Gentium Plus;10;400;;normal;;LTR
T	11 w=3.1689	(()
Mx 	223.2860
T	43 w=6.5381	(H)
Mx 	229.8241
T	12 w=3.1689	())
Mx 	257.3480
T	27 19 17 28 w=16.3672	(80.9)
Mx 	273.7152
T	131 w=3.5791	(°)
Mx 	343.0677
T	27 22 17 21 w=16.3672	(83.2)
Mx 	359.4349
T	131 w=3.5791	(°)
Mx 	190.4882
My 	97.4402
Draw line	190.4882	103.8816	42.8598	0.8000
Mx 	214.4882
Set font 	Gentium Plus;10;400;Italic;normal;;LTR
T	478 w=4.6289	(β)
Mx 	220.1171
Set font 	Gentium Plus;10;400;;normal;;LTR
T	11 w=3.1689	(()
Mx 	223.2860
T	48 88 w=13.3594	(Mu)
Mx 	236.6454
T	12 w=3.1689	())
Draw line	233.3480	103.8816	85.7197	0.8000
Mx 	257.3480
T	27 25 17 26 w=16.3672	(86.7)
Mx 	273.7152
T	131 w=3.5791	(°)
Draw line	319.0677	103.8816	85.7197	0.8000
Mx 	343.0677
T	27 26 17 26 w=16.3672	(87.7)
Mx 	359.4349
T	131 w=3.5791	(°)
Mx 	190.4882
My 	128.0359
Draw line	190.4882	115.8816	42.8598	0.8000
Draw line	190.4882	134.4773	42.8598	0.4000
Draw line	233.3480	115.8816	85.7197	0.8000
Draw line	233.3480	134.4773	85.7197	0.4000
Mx 	257.3480
T	43 w=6.5381	(H)
Mx 	263.8861
T	11 w=3.1689	(()
Mx 	267.0551
T	48 88 w=13.3594	(Mu)
Mx 	280.4144
T	12 w=3.1689	())
Mx 	286.2275
T	14 w=4.1016	(+)
Mx 	292.9733
T	41 w=4.7803	(F)
Mx 	297.7535
My 	124.5393
Set font 	Gentium Plus;6.421;400;;normal;;LTR
T	21 w=3.0130	(2)
Draw line	319.0677	115.8816	85.7197	0.8000
Draw line	319.0677	134.4773	85.7197	0.4000
Mx 	343.0677
My 	128.0359
Set font 	Gentium Plus;10;400;;normal;;LTR
T	43 w=6.5381	(H)
Mx 	349.6058
T	11 w=3.1689	(()
Mx 	352.7748
T	48 88 w=13.3594	(Mu)
Mx 	366.1341
T	12 w=3.1689	())
Mx 	371.9469
T	14 w=4.1016	(+)
Mx 	378.6922
T	38 79 w=8.0713	(Cl)
Mx 	386.7635
My 	124.5393
Set font 	Gentium Plus;6.421;400;;normal;;LTR
T	21 w=3.0130	(2)
Mx 	190.4882
My 	146.6316
Mx 	214.4882
Set font 	Gentium Plus;10;400;Italic;normal;;LTR
T	478 w=4.6289	(β)
Mx 	220.1171
Set font 	Gentium Plus;10;400;;normal;;LTR
T	11 w=3.1689	(()
Mx 	223.2860
T	43 w=6.5381	(H)
Mx 	229.8241
T	12 w=3.1689	())
Mx 	257.3480
T	27 19 17 28 w=16.3672	(80.9)
Mx 	273.7152
T	131 w=3.5791	(°)
Mx 	343.0677
T	27 22 17 21 w=16.3672	(83.2)
Mx 	359.4349
T	131 w=3.5791	(°)
Mx 	190.4882
My 	165.2273
Draw line	190.4882	171.6687	42.8598	0.8000
Mx 	214.4882
Set font 	Gentium Plus;10;400;Italic;normal;;LTR
T	478 w=4.6289	(β)
Mx 	220.1171
Set font 	Gentium Plus;10;400;;normal;;LTR
T	11 w=3.1689	(()
Mx 	223.2860
T	48 88 w=13.3594	(Mu)
Mx 	236.6454
T	12 w=3.1689	())
Draw line	233.3480	171.6687	85.7197	0.8000
Mx 	257.3480
T	27 25 17 26 w=16.3672	(86.7)
Mx 	273.7152
T	131 w=3.5791	(°)
Draw line	319.0677	171.6687	85.7197	0.8000
Mx 	343.0677
T	27 26 17 26 w=16.3672	(87.7)
Mx 	359.4349
T	131 w=3.5791	(°)
End page
Finish
